### PR TITLE
controller/tests: Avoid hardcoding SSH MC offsets

### PR DESCRIPTION
**Pull request by cgwalters**
In prep for adding more.

### CODE REVIEW

File: pkg/controller/template/render_test.go [hunk 1]
```diff
@@ -316,6 +316,12 @@ func TestGenerateMachineConfigs(t *testing.T) {
 	}
 }
 
+func validateSSHConfig(t *testing.T, mc *mcfgv1.MachineConfig) {
```
Comment (kikisdeliveryservice) [inline pkg/controller/template/render_test.go]: I would move this down with the other helper functions in the bottom of file to stay consistent with file conventions.

**Comment by kikisdeliveryservice** [comment]
Thanks @cgwalters  Is this prep for osImageUrl tests?

**Comment by ashcrow** [comment]
It's now erroring trying to make/reference release images. I'll kick it off again shortly.

**Pull request merged**